### ISSUE TITLE
improve README.md organization

### ISSUE THREAD

**Issue by honzi** [iterami/UnitConverter.htm#6]
As more and more units are added, the `README.md` file is getting quite cumbersome. It should be reorganized to better display available units in a smaller space.

**Comment by cbsorrilha** [comment]
I suggest use the readme to show installation instructions and add a markdown file to each type of unit. Something like:

```
acceleration.md
angle.md
area.md
...
```

And so on. And point link to them on the readme on a section called "Converts To".

What do you think?

**Comment by honzi** [comment]
Thanks for your suggestion. That's definitely one possible approach.

I currently prefer to keep all README related things in one file for organizational purposes, as splitting it up into multiple files could become more trouble than it's worth (such as requiring additional clicks to find information and increasing project bloat). I do like the idea of adding installation information, as that is something `iterami` READMEs lack.

I haven't updated any issues lately regarding much needed README updates, but I have been thinking of a few options:
1. Move units into a multi-column table with one unit per table cell. That would greatly shorten the vertical length of the `README.md` file, but potentially make it more difficult to glance over. That might not be a big issue, as simply having the units listed within the README may be sufficient.
2. Don't provide a list of converted units. I don't like this one much, but listing the units within `<select>`s gives users the same amount of information as a list in the README.
3. Redo all `iterami` `README.md` files to only include repository descriptions and useful links, and move all other information (such as installation guides, lists of units, etc) to a separate `iterami` repository to standardize information in one location.

At the moment I like option 3 the most, but I've been distracted by other ideas so I haven't even started that process. I'm always open to ideas and thoughts if you have more.

**Comment by cbsorrilha** [comment]
I've suggested that approach thinking in the readme as a location to explain the project & the proceedures to get it running. But you're right, too much clicks will be needed for the user to see the units available.

What you think about abstracting it from the readme and putting it on a presentation page on ghpages?

I think that the third option is the best two. But it will demand much time, I think.

**Comment by honzi** [comment]
I've thought about it a lot more, and I've decided to go with a central repository to contain all the documentation, so I made https://github.com/iterami/Documentation.htm. It'll allow storage of more detailed information about specific repositories and also have generic files like `CONTRIBUTING.md` to be shared between repositories. This will mean that I'll simplify repository `README.md` files to mostly only include links to more detailed information.

It'll take a lot of time, but I think it will be worth it. Thanks for reminding me to take care of this! I'm going to close this issue for now because the `README.md` file seems sufficiently organized.
